ui/src/components/overview/MobileClientBuildList.js

```diff
@@ -29,6 +30,10 @@ class MobileClientBuildList extends Component {
                                             created <Moment fromNow>{mobileClientBuild.status.startTimestamp}</Moment>
                                     </span>
                                 </Col>
+                                <Col md={12}>
+                                    <BuildDownloadComponent downloadURL={mobileClientBuild.downloadURL} QRSize={256}/>
+                                </Col>
+                                
```

Comment:
We can remove these lines and the import at the top altogether, I will integrate this component on my branch.